mne/preprocessing/ssp.py

```diff
@@ -150,9 +150,19 @@ def compute_proj_ecg(raw, raw_event=None, tmin=-0.2, tmax=0.4,
                      tstart=0., qrs_threshold='auto', filter_method='fir',
                      iir_params=None, copy=True, return_drop_log=False,
                      meg='separate', verbose=None):
-    """Compute SSP/PCA projections for ECG artifacts.
+    """Compute SSP (signal-space projection) vectors for ECG artifacts.
 
-    .. note:: raw data will be loaded if it is not already.
+    This function will:
+
+      - filter the ECG data channel,
+      - find ECG R wave peaks using :func:`mne.preprocessing.find_ecg_events`,
+      - filter the raw data,
+      - create `~mne.Epochs` around the R wave peaks, capturing the heartbeats,
+      - optionally average the `~mne.Epochs` to produce an `~mne.Evoked` if
+        ``average=True`` was passed (default), and finally
+      - calculate SSP projection vectors on that data to capture the artifacts.
```

Comment:
these are indented too far, they're getting rendered as both bullets and blockquotes.

https://23991-1301584-gh.circle-artifacts.com/0/dev/generated/mne.preprocessing.compute_proj_ecg.html#mne.preprocessing.compute_proj_ecg

Reply:
I have fixed that, and also changed the list to an enumeration.